shell/platform/windows/angle_surface_manager.cc

```diff
@@ -200,7 +204,11 @@ void AngleSurfaceManager::CleanUp() {
   }
 
   if (egl_display_ != EGL_NO_DISPLAY) {
-    eglTerminate(egl_display_);
+    // Display is reused between instances so only terminate display
+    // if destroying last instance
+    if (instance_count_ == 1) {
```

Comment:
as defense in depth, worth setting count to 0 when you detect this?

Reply:
Not sure I see the point. It is decremented to zero in destructor right after CleanUp returns. This all happens on platform thread so there shouldn't be any concurrency issues.